include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -705,7 +743,7 @@ class argument_parser
                  detail::format_html,
                  detail::format_man,
                  detail::format_copyright/*,
-                 detail::format_ctd*/> format{detail::format_help(0)};
+                 detail::format_ctd*/> format{detail::format_help{{}, false}};
```

Comment:
Does this need to be initialised?

Reply:
When the constructor is called the format would be default initialised which is not possible because format_parse has no default constructor..?

Reply:
Well, it's weird because it suggests that it has some kind of special meaning. 

Why not use std::monostate which was created for this purpose?
https://en.cppreference.com/w/cpp/utility/variant/monostate

Reply:
uh I did not know of this :+1:

Reply:
too bad... If I use std::monostate the std::visit call fails because monostate obviously has no `add_flag` member function :/

Reply:
Hm, than maybe at least write something like "The state will be overwritten in any case." as comment?

Reply:
Still open.

Reply:
ok fine